dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -46,12 +47,20 @@ void NotificationsLongRunningPlatformImpl::WaitForWinMainEvent()
     m_shutdownTimerManager->Wait();
 }
 
-// Example of one function. We will add more as we need them.
-STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::RegisterFullTrustApplication(_In_ PCWSTR /*processName*/, _In_ GUID /*remoteId*/, _Out_ GUID* /*appId*/) noexcept
+
+STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::RegisterFullTrustApplication(
+    _In_ PCSTR processName, GUID remoteId, _Out_ LPWSTR* appId) noexcept
 {
     auto lock = m_lock.lock_shared();
-    RETURN_HR_IF(WPN_E_PLATFORM_UNAVAILABLE, m_shutdown);
-    
-    return E_NOTIMPL;
-}
 
+    UNREFERENCED_PARAMETER(processName);
```

Comment:
Seems like you are planning to address this as part of your Registry work?

In the PR description, be very clear about the scope of this PR and what will be addressed later.